p2p/src/authenticated/actors/tracker/ingress.rs

```diff
@@ -98,6 +98,18 @@ impl<E: Spawner + Metrics, C: Verifier> Mailbox<E, C> {
         rx.await.unwrap()
     }
 
+    pub fn try_release(&mut self, peer: C::PublicKey) -> bool {
+        let Err(e) = self.sender.try_send(Message::Release { peer }) else {
+            return true;
+        };
+        if e.is_full() {
+            return false;
+        }
+
+        // If any other error occurs, we should panic!
+        panic!("Unexpected error while trying to send message: {:?}", e);
```

Comment:
This is just to maintain the current behavior...there is a larger conversation to be had about changing this to just "drop" if disconnected.